### ISSUE TITLE
Test data generation for function calling

### ISSUE THREAD

**Issue by alexHeu** [explodinggradients/ragas#1313]
Hey,
I was wondering if you think it would be possible to create a synthetic dataset for function calling tasks?
I would like to use that dataset for a finetuning experiment.

Thanks for any guidance that you might have!

**Comment by anistark** [comment]
@alexHeu No, Ragas currently does NOT support generating synthetic datasets for function calling/tool use tasks. Ragas synthetic data generation is limited to RAG-style Q&A scenarios.
